Brokerages/InteractiveBrokers/InteractiveBrokersBrokerage.cs

```diff
@@ -1647,6 +1649,13 @@ public IEnumerable<BaseData> GetNextTicks()
 
             foreach (var tick in ticks)
             {
+                // if we have a cached data time zone for this symbol, perform the conversion 
+                DateTimeZone dataTimeZone;
+                if (_dataTimeZones.TryGetValue(tick.Symbol.Value, out dataTimeZone))
+                {
+                    tick.Time = tick.Time.ConvertTo(TimeZones.NewYork, dataTimeZone);
```

Comment:
My only concern here was to make sure we are emitting data on the exchange time zone. If that's already covered elsewhere, then no worries, but it looks like we're emitting  data in the data time zone which is incorrect.  Data time zone is for reading raw data and then it should be immediately converted into the exchange time zone for consumption by the algorithm